Framework/Core/src/DeviceSpecHelpers.cxx

```diff
@@ -708,14 +722,16 @@ void DeviceSpecHelpers::processInEdgeActions(std::vector<DeviceSpec>& devices,
   for (size_t edge : inEdgeIndex) {
     auto& action = actions[edge];
 
-    size_t consumerDevice;
+    size_t consumerDevice = -1;
```

Comment:
Is this necessary with `else` branch added?

Reply:
Not necessary, but allows us to spot actual issues if the code is changed.